bower.json

```diff
@@ -29,7 +29,9 @@
     "leaflet-switch-scale-control": "0.1.1-beta.1",
     "leaflet.zoomslider": "Flexberry/Leaflet.zoomslider#leaflet-1.2.0",
     "leaflet-minimap": "3.4.0",
-    "osmtogeojson": "2.2.12"
+    "osmtogeojson": "2.2.12",
+    "leaflet-omnivore": "0.3.3",
+    "terraformer": "^1.0.8"
```

Comment:
better to use strict version, we already had problems with html2canvas and ember-data

Reply:
fix